server/execution/entities/execution.py

```diff
@@ -28,6 +30,28 @@ def __init__(self, id: int, name: str, scenario: Scenario,
         self.starting_time = starting_time
         self.notifications = []
 
+    def start_execution(self):
+        """ Performs the execution start protocol. """
+        if not self.scenario:
+            raise InternalServerError("Execution without a scenario detected.")
```

Comment:
In general, who should be responsible for having valid scenarios/executions? The scenario admin interface?

Reply:
Noteable for Monday Meeting. I would not decide that in this PR.